.github/workflows/build.yaml

```diff
@@ -24,6 +36,8 @@ jobs:
           - runtime: windows
             os: windows-latest
             reportName: windows-test-report
+    env:
+      USE_LOCAL_PLUGIN: ${{ inputs.useLocalPlugin || 'false' }}
```

Comment:
Wondering if there's something I'm missing here. Normally a boolean expression of the form `X || false` is always equal to `X`. But then I noticed `'false'` is quoted like a string and in some scripting languages using a non-null string in a boolean expression is equal to `true`.

Reply:
See definition of literals here: https://docs.github.com/en/actions/learn-github-actions/expressions#literals and the statement: 

> Note that in conditionals, falsy values (false, 0, -0, "", '', null) are coerced to false and truthy (true and other non-falsy values) are coerced to true.

It would seem that `'false'` in a conditional statement would be coerced to `true`.

Reply:
Got it. I have changed it to boolean.